README.md

```diff
@@ -78,9 +129,13 @@ By default the DIVA reference implementation runs with `in memory` session manag
 
 The MyHome example in the [diva-js-reference-3p-frontend](https://github.com/Alliander/diva-js-reference-3p-frontend) frontend displays a map image that uses the Bing maps API through this backend. To make it work, provide your own private bing maps api key as an export, or set a placeholder (of course then the Bing API won't work correctly). A Bing maps key can be obtained from [Bing](https://msdn.microsoft.com/en-us/library/ff428642.aspx).
 
+## Logging
+
+Both this reference implementation as well as diva-irma-js use [log4js](https://www.npmjs.com/package/log4js) as a logger. By default no logs are printed. By setting `LOG_LEVEL` and `DIVA_LOG_LEVEL` (or by setting their values in `src/config.js`), both this reference implementation and the Diva library can be set to log events to stdout. See [here](https://www.npmjs.com/package/log4js#usage) for the available log levels, where TRACE will provide the most fine-grained debug statements.
+
 ## Tests
 
-There are currently no tests for the backend.
+The tests for the backend require a working and configured IRMA Api Server, which is hard to do in a CI-environment. Therefore, tests aren't run yet automatically.
```

Comment:
`automatically yet`

Reply:
To some degree, basic tests for the backend are possible. We could also create a mock client, but that would require hooking in to irmago for the client part. Scanning QR codes is hard to test and brittle in CI.